I get a linking issue at the end of the compilation (gcc 4.9.2 dev):

```
Linking CXX executable OpenDungeons.bin
CMakeFiles/OpenDungeons.bin.dir/source/MenuModeMultiplayer.cpp.o: In function `MenuModeMultiplayer::serverButtonPressed()':
MenuModeMultiplayer.cpp:(.text+0x891): undefined reference to `std::basic_string<char, std::char_traits<char>, std::allocator<char> > boost::locale::conv::to_utf<char>(char const*, char const*, std::string const&, boost::locale::conv::method_type)'
CMakeFiles/OpenDungeons.bin.dir/source/ODFrameListener.cpp.o: In function `ODFrameListener::notifyChatChar(int)':
ODFrameListener.cpp:(.text+0xe73): undefined reference to `std::basic_string<char, std::char_traits<char>, std::allocator<char> > boost::locale::conv::to_utf<char>(char const*, char const*, std::string const&, boost::locale::conv::method_type)'
collect2: error: ld returned 1 exit status
CMakeFiles/OpenDungeons.bin.dir/build.make:3573: recipe for target 'OpenDungeons.bin' failed
make[2]: *** [OpenDungeons.bin] Error 1
CMakeFiles/Makefile2:60: recipe for target 'CMakeFiles/OpenDungeons.bin.dir/all' failed
make[1]: *** [CMakeFiles/OpenDungeons.bin.dir/all] Error 2
Makefile:137: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -284,7 +284,7 @@ find_package(OpenAL REQUIRED)
 
 if(UNIX)
     find_package(Threads REQUIRED)
-    find_package(Boost COMPONENTS thread system filesystem REQUIRED)
+    find_package(Boost COMPONENTS thread system filesystem locale REQUIRED)
 endif()
 
 ##################################

